src/controllers/metadata.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+const _ = require('lodash')
+const config = require('../services/config')
+
+const baseURI = config.getIn(['server', 'base_uri'])
+const urls = _.mapValues({
+  health_url: '/health',
+  transfer_url: '/transfers/:id',
+  transfer_fulfillment_url: '/transfers/:id/fulfillment',
+  transfer_state_url: '/transfers/:id/state',
+  connectors_url: '/connectors',
+  accounts_url: '/accounts',
+  account_url: '/accounts/:name',
+  subscription_url: '/subscriptions/:id',
+  subscription_notification_url: '/subscriptions/:subscription_id/notifications/:notification_id'
+}, function (path) { return baseURI + path })
```

Comment:
Does it have to be the full URL? Relative URLs tend to be simpler because they don't have issues with proxies etc. Also, do you think it would be better to keep the URLs in their own sub-object?